src/main_app/db/services/db_service.py

```
@@ -0,0 +1,49 @@
+        if record_id is None:
+            return False
+
+        try:
+            record = db.session.get(self.model, record_id)
+            if not record:
+                return False
+
+            db.session.delete(record)
+            db.session.commit()
+            return True
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)